python-mini-projects/projects/Scraping Medium Articles/scraping_medium.py

```diff
@@ -0,0 +1,71 @@
+import os
+import sys
+import requests
+import re
+from bs4 import BeautifulSoup
+
+# switching to current running python files directory
+os.chdir('\\'.join(__file__.split('/')[:-1]))
+
+# function to get the html of the page
+def get_page():
+	global url
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Global variables are dangerous because they can be simultaneously accessed from multiple sections of a program. This frequently results in bugs. Most bugs involving global variables arise from one function reading and acting on the value of a global variable before another function has the chance to set it to an appropriate value. We recommend using a configuration module to mutate global state.

[Learn more](https://docs.python.org/3/faq/programming.html#how-do-i-share-global-variables-across-modules)

Similar issue at line number 36.